AzureCalling/External/Calling/TableViewDataSource.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class TableViewDataSource: NSObject, UITableViewDataSource {
+
+    // MARK: Properties
+
+    private var cellViewModelDataArray: [BottomDrawerCellViewModel]
```

Comment:
To keep things simpler, I don't think we need the abstraction of `BottomDrawerCellViewModel`. We can just have an array of "AudioDeviceOption" objects.

Reply:
The idea behind his was for it to be compatible with the participant list, which will also make use of BottomDrawerCellViewModel.